src/index.ts

```diff
@@ -87,6 +87,7 @@ cli.option('-w, --watch [...files]', 'watch files', { default: false })
     .option('--format [format]', 'choose one from "cjs" or "esm"')
     .option('--external [...external]', 'external')
     .option('--no-auto-external', 'if not found module, auto as external', { default: true })
+    .option('--sourcemap', 'generate sourcemap or not')
```

Comment:
when we need to receive more than just boolean, we should use `[]` or `<>` to indicate that we need to receive a value.

```suggestion
    .option('--sourcemap [sourcemap]', 'generate sourcemap or not')
```

in this case, we may receive a string of true or false, which you need to handle specially

Reply:
> in this case, we may receive a string of true or false, which you need to handle specially

```ts
// disable generate
farmup --sourcemap=false // args.sourcemap "false" => false
farmup // args.sourcemap undefined, use farm's default value
// generate
farmup --sourcemap // args.sourcemap true
farmup --sourcemap=true // args.sourcemap "true" => true
farmup --sourcemap=all // args.sourcemap "all"
```